op-guide/build.md

```diff
@@ -0,0 +1,145 @@
+# Build
+
+## Supported platforms
+
+The following table lists TiDB support for common architectures and operating systems. 
+
+|Architecture|Operating System|Status|
+|------------|----------------|------|
+|amd64|Linux Ubuntu (14.04+)|Stable|
+|amd64|Linux CentOS (7+)|Stable|
+|amd64|Mac OSX|Experimental|
+
+## Prerequisites
+
++ Go [1.5+](https://golang.org/doc/install)
++ Rust [nightly version](https://www.rust-lang.org/downloads.html)
++ GCC 4.8+ with static library
+
+### Install GO
+
+#### Linux
+
+```bash
+curl -L https://storage.googleapis.com/golang/go1.6.3.linux-amd64.tar.gz -o golang.tar.gz
+tar -C /usr/local -xzf golang.tar.gz
+```
+
+#### Mac OS X
+
+```bash
+curl -L https://storage.googleapis.com/golang/go1.6.3.darwin-amd64.tar.gz -o golang.tar.gz
+tar -C /usr/local -xzf golang.tar.gz
+```
+
+### Install Rust
+
+```bash
+curl -sSf https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly
+```
+
+### Install GCC
+
+#### Linux Ubuntu
+
+```bash
+apt-get update
+apt-get install -y gcc g++
+```
+
+#### Linux CentOS
+
+```bash
+yum install -y gcc-c++ glibc-static libstdc++-static
+```
+
+#### Mac OS X
+
+```bash
+xcode-select --install
+brew update
+brew tap homebrew/versions
+brew install gcc48 --use-llvm
+```
+
+## Build TiDB project
+
+```
+# Create a root path for building and installing TiDB. 
+mkdir -p /Users/tidb
+export TIDB_PATH=/Users/tidb
+# All the binaries are installed in the `bin` directory. 
+mkdir -p $TIDB_PATH/bin
+```
+
+### Build TiDB
+
+```bash
+# go is required
+go version
+# go version go1.6 darwin/amd64
+
+# GOPATH should be set correctly.
+export GOPATH=$TIDB_PATH/go
+
+git clone https://github.com/pingcap/tidb.git $GOPATH/src/github.com/pingcap/tidb
+cd $GOPATH/src/github.com/pingcap/tidb
+
+make server
+cp -f ./tidb-server/tidb-server $TIDB_PATH/bin
+cd $TIDB_PATH
+```
+
+### Build PD
+
+```bash
+export GOPATH=$TIDB_PATH/go
+
+git clone https://github.com/pingcap/pd.git $GOPATH/src/github.com/pingcap/pd
+cd $GOPATH/src/github.com/pingcap/pd
+
+make build
+cp -f ./bin/pd-server $TIDB_PATH/bin
```

Comment:
Maybe we should also copy the `templates` to $TIDB_PATH if we want to see the balance events.